libs/partners/openai/langchain_openai/chat_models/base.py

```diff
@@ -1281,6 +1281,28 @@ class GetPopulation(BaseModel):
               'args': {'location': 'New York, NY'},
               'id': 'call_6ghfKxV264jEfe1mRIkS3PE7'}]
 
+        Note that ``openai >= 1.32`` supports a ``parallel_tool_calls`` parameter
+        that defaults to ``True``. This parameter can be set to ``False`` to
+        disable parallel tool calls:
+
+        .. code-block:: python
+
+            ai_msg = llm_with_tools.invoke(
+                "What is the weather in LA and NY?",
+                parallel_tool_calls=False,
+            )
+            ai_msg.tool_calls
+
+        .. code-block:: python
+
+            [{'name': 'GetWeather',
+            'args': {'location': 'Los Angeles, CA'},
+            'id': 'call_4OoY0ZR99iEvC7fevsH8Uhtz'}]
+
+        Like other runtime parameters, ``parallel_tool_calls`` can be bound to a model
+        using ``llm.bind(parallel_tool_calls=False)`` or during instantiation by
```

Comment:
imo this should be with bind_tools (feels like you'd almost always do this at time that you're binding the tools)